src/app/clients/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import React, { useState } from "react";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+import { ClientsTableRow } from "@/app/clients/getClientsTableData";
+import Paper from "@mui/material/Paper/Paper";
+import Alert from "@mui/material/Alert";
+import Statuses from "@/app/clients/ActionBar/Statuses";
+import Actions from "@/app/clients/ActionBar/Actions";
+
+interface Props {
+    selected: number[];
+    data: ClientsTableRow[];
+}
+
+const OuterDiv = styled.div`
+    display: flex;
+    padding: 1rem;
+    gap: 0.5rem;
+    border-radius: 0.5rem;
+    background-color: ${(props) => props.theme.main.background[5]};
+`;
+
+const AlertBox = styled.div`
+    padding: 0.5rem 1rem 1rem;
+    border-radius: 0.5rem;
+`;
+
+const StyledPaper = styled(Paper)`
+    margin: 1rem;
+`;
+
+const ActionBar: React.FC<Props> = ({ selected, data }) => {
+    const [statusAnchorElement, setStatusAnchorElement] = useState<HTMLElement | null>(null);
+    const [actionAnchorElement, setActionAnchorElement] = useState<HTMLElement | null>(null);
+
+    const [modalError, setModalError] = useState<string | null>(null);
+
+    return (
+        <StyledPaper>
+            <Statuses
```

Comment:
For these you might just be able to do {...props}? Not sure tho

Reply:
<3